src/main/java/frc/robot/maps/RobotMap.java

```diff
@@ -153,4 +93,34 @@ public LiftMap getLiftMap() {
         return new LiftMap();
     }
 
+    public static class IndexMap {
+        public SendableSpeedController singulator() {
```

Comment:
Wait, the same functions are also in the `IntakeMap` above. What's up with that?